src/content/news/update-v1-3.md

```
@@ -0,0 +1,118 @@
+---
+
+The Third update of the game has been released. It includes **quite a few** of balance changes and three new characters - **Edel**, **Wirbel** and **Flamme**!
+
+---
+
+## New characters
+
+This patch brings three new characters to the game.
+
+-   **Edel**:
+    -   Edel's a control character, who makes use of her hypnosis to prevent the opponent from enacting their plan.
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)